charts/ocis/values.yaml

```diff
@@ -197,6 +199,22 @@ features:
     publiclink:
       # -- Enforce a password on writable public link shares.
       writeableShareMustHavePassword: false
+    # automatically accept incoming shares
+    autoAcceptShares: true
```

Comment:
just interesting, why value is true? 
we use false by default. https://doc.owncloud.com/ocis/next/deployment/services/s-list/frontend.html#auto-accept-shares

Reply:
Shares auto accept should be set to true on default. We should also set it to true in the ocis single binary. But there are problems with the ci-tests because they try to accept the shares and run into failures. We need to adjust the testsuite first.

Reply:
I can adapt api and e2e test. do we have ticket for that?

Reply:
Yes, there is actually: https://github.com/owncloud/ocis/issues/7241